Resources/Prototypes/Catalog/Fills/Crates/fun.yml

```diff
@@ -172,3 +172,18 @@
           amount: 1
         - id: VehicleKeySyndicateSegway
           amount: 1
+
+- type: entity
+  id: CrateFunBoxing
+  parent: CrateGenericSteel
+  components:
+  - type: StorageFill
+    contents:
+      - id: ClothingHandsGlovesBoxingRed
+      - id: ClothingHandsGlovesBoxingBlue
+      - id: ClothingHandsGlovesBoxingYellow
+      - id: ClothingHandsGlovesBoxingGreen
```

Comment:
Aren't these combat items, I don't think they should be orderable.

Reply:
What do you mean by combat items? Like, yeah, they meant for boxing and have reduced brute and increased stam damage, but they're not OP or anything

Reply:
idk why these wouldnt be fine

Reply:
In my brain I thought it was boxingcomp.